language-pack-fr/data/messages/breach.dat

```diff
@@ -0,0 +1,11 @@
+{
+	messages: [

```

Comment:
Please put quotes around JSON properties. Our JSON serializer can handle this, but it makes life easier and JSON linting possible.

E.g.
```json
{
    "messages": []
}
```